README.md

```diff
@@ -1,52 +1,124 @@
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+  </ol>
+</details>
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+Our goal is to create a CMS for everyone in an organization. It's our belief that a CMS should inspire every one to express themselves on the web by providing a modern and flexible developer experience and an intuitive interface for content editors. ApostropheCMS is built for teams, and we want to make your job more fun.
+
+#### Built With
+
+* [Node](https://nodejs.org/en/)
+* [Nunjucks](https://mozilla.github.io/nunjucks/)
+* [MongoDB](https://www.mongodb.com/)
+
+<!-- GETTING STARTED -->
 ## Getting Started
 
-The [ApostropheCMS documentation](https://docs.apostrophecms.org/apostrophe/) will walk you through some simple steps to get your [local environment set up](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/setting-up-your-environment) with the technology stack that runs ApostropheCMS (Node.js, MongoDB, ImageMagick) along with installing ApostropheCMS itself.
+To get started with ApostropheCMS, follow these steps to set up a local development environment.
+
+#### Prerequisites
+
+We recommend installing the following with [Homebrew](https://brew.sh/) on macOS. If you're on Linux, you should use your package manager (apt or yum). If you're on Windows, we recommend the Windows Subsystem for Linux.
+
+| Software | Minimum Version | Notes
+| ------------- | ------------- | -----
+| Git  | Any
+| Xcode  | Current | Only needed on macOS
+| Node.js | 8.x | Or better
+| npm  | 6.x  | Or better
+| MongoDB  | 3.6  | Or better
+| Imagemagick (optional)  | Any | GIF support, faster image uploads
 
-Then [create a new project](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/creating-your-first-project) on your computer and work on building your beautiful website.
 
-The ApostropheCMS documentation will walk you through [editing page templates](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/editing-page-templates), [global content](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/global), [creating custom widgets](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/custom-widgets) and [navigation](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/building-navigation), along with [configuring reusable "pieces"](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/reusable-content-with-pieces) and [global settings](https://docs.apostrophecms.org/apostrophe/core-concepts/global-settings/settings). Plus, a whole lot more!
+#### New Project
+Our recomended way to start a new project with ApostropheCMS is to use the [Apostrophe CLI](https://github.com/apostrophecms/apostrophe-cli). Alternatively, you can simply clone this repo and `npm install`.
```

Comment:
What you copied here is appropriate for the boilerplate or apostrophe-open-museum. If you try to just git clone the apostrophe core repo and "npm install", you don't get a website.

Reply:
TL;DR just kill this "alternatively" sentence, it won't work in this context.

Reply:
I will link to the boilerplate here, I think that's reasonable